extensions/amp-brightcove/0.1/test/validator-amp-brightcove.out

```diff
@@ -1,4 +1,5 @@
 FAIL
 amp-brightcove/0.1/test/validator-amp-brightcove.html:31:2 The tag 'amp-brightcove' requires including the 'amp-brightcove' extension JavaScript. (see https://www.ampproject.org/docs/reference/components/amp-brightcove) [AMP_TAG_PROBLEM]
```

Comment:
Lets include the amp-brightcove extension in the html since there are supposed to be valid uses in the test file.
`<script async custom-element="amp-brightcove" src="https://cdn.ampproject.org/v0/amp-brightcove-0.1.js"></script>`